CONTRIBUTING.md

```diff
@@ -55,19 +58,21 @@ To set up `pysatMissions` for local development:
    ``test_``.
 
 4. When you're done making changes, run all the checks to ensure that nothing
-   is broken on your local system::
-
-    pytest -vs
+   is broken on your local system:
+   ```
+   pytest -vs
+   ```
 
 5. Update/add documentation (in ``docs``), if relevant
 
-5. Commit your changes and push your branch to GitHub::
-
+6. Commit your changes and push your branch to GitHub:
+   ```
     git add .
     git commit -m "Brief description of your changes"
```

Comment:
This should use the recommended commit styling, and also note that you don't need to push every time you commit.

Reply:
We should update this throughout the projects.  Can't find a short document with the style recommendations.  Was it numpy or scipy that defined this?

Reply:
It was numpy, but they may have "updated it away" in their current docs.  An older version of the docs would work just fine for us.

Reply:
Found it:  https://numpy.org/doc/stable/dev/development_workflow.html